src/include/storage/store/table_statistics.h

```diff
@@ -34,8 +49,20 @@ class TableStatistics {
         numTuples = numTuples_;
     }
 
+    inline PropertyStatistics& getPropertyStatistics(common::property_id_t propertyID) {
```

Comment:
Add assertion here that the map contains propertyID.

Reply:
Yeah. But the thing is that right now when we're trying to debug, it's not easy to trace where the exception by `at` was thrown out. `assert` can help in this case.

Reply:
Fair enough